list_test.go

```diff
@@ -301,14 +302,14 @@ func TestListsItemsIP(t *testing.T) {
 	want := []ListItem{
 		{
 			ID:         "2c0fc9fa937b11eaa1b71c4d701ab86e",
-			IP:         "192.0.2.1",
+			IP:         pointer.ToString("192.0.2.1"),
```

Comment:
once the dependency is cleaned up, we can use the following

```suggestion
			IP:         cloudflare.StringPtr("192.0.2.1"),
```